python/TestHarness/RunParallel.py

```diff
@@ -148,6 +149,16 @@ def startReadyJobs(self, slot_check):
       os.chdir(saved_dir)
       sys.path.pop()
 
+  def getOutputFromFiles(self, tester):
+    # TODO: Use redirect_stdout when ready (libMesh update required Ref: https://github.com/libMesh/libmesh/issues/1159)
+    # file_list = glob.glob(os.path.join(tester.specs['test_dir'], tester.specs['redirect_stdout'] + '.*'))
+    file_list = glob.glob(os.path.join(tester.specs['test_dir'], 'stdout.processor.*'))
+    stdout = ''
```

Comment:
I'm being pedantic here, but I'd name this file_output or something else. "stdout" means stdout to me and this is actually stdout + stderr.